### ISSUE TITLE
[Remote-SSH Bug]: 

### ISSUE THREAD

**Issue by Zilong-Yuan** [microsoft/vscode-remote-release#8936]
### Is there an existing issue for this bug?

- [X] I have searched the existing issues

### Required Troubleshooting Steps

- [X] I have followed these troubleshooting steps
- [X] I have tried both values of the `remote.SSH.useLocalServer` setting

### Connect Locally

It connects successfully

### ->

no output log and stuck at  Downloading....
as soon as update the latest version of VSCODE
![image](https://github.com/microsoft/vscode-remote-release/assets/75056948/9a5365f2-4808-429d-b6c3-a28b7aaa7b10)

### Expected Behavior

it usually finishes in a minite a few updates before

### Steps To Reproduce

just  update to the latest VSCODE and know it does not work

### Remote-SSH Log

<details>
<summary>Remote-SSH Log</summary>
<p>

```
[PASTE LOG HERE]
```

</p>
</details>
no output

### Anything else?

![image](https://github.com/microsoft/vscode-remote-release/assets/75056948/051f8fe6-64ce-44a5-a4dc-d45f752c1236)

**Comment by rhawleyiii** [comment]
I am seeing this same behavior today.

**Comment by alexmcchao** [comment]
I am seeing the same issue too

**Comment by eufmike** [comment]
Same here. I started a fresh .vscode-server folder. It downloaded "code-8b617bd08fd9e3fc94d14adb8d358b56e3f72314" then stuck there. These are what I got. 
```shell
     151 Sep  8 12:34 .                                                    
     563 Sep  8 12:33 ..                                                   
      25 Sep  8 12:34 cli                                                  
     413 Sep  8 12:34 .cli.8b617bd08fd9e3fc94d14adb8d358b56e3f72314.log    
17981560 Sep  6 14:42 code-8b617bd08fd9e3fc94d14adb8d358b56e3f72314        
```

**Comment by rhawleyiii** [comment]
There's another bug report open for this issue.  The recommended steps linked below worked for me, but before you use them be sure to kill any running vscode processes on the remote server, and remove the .vscode-server folder.

https://github.com/microsoft/vscode-remote-release/issues/8928#issuecomment-1711316170

```
$ commit_id="8b617bd08fd9e3fc94d14adb8d358b56e3f72314"
$ wget https://vscode.cdn.azure.cn/stable/${commit_id}/vscode-server-linux-x64.tar.gz
$ mkdir -p "~/.vscode-server/cli/servers/Stable-${commit_id}/server"
$ tar xf vscode-server-linux-x64.tar.gz --strip 1 -C "~/.vscode-server/cli/servers/Stable-${commit_id}/server"
```

**Comment by roblourens** [comment]
Sorry for the trouble, this should be fixed in 0.106.2

**Comment by sayajin101** [comment]
What worked for me was on the `Remote - SSH` extension clicking on the 'Switch to Pre-Release Version` in the mean time

**Comment by Zilong-Yuan** [comment]
> What worked for me was on the `Remote - SSH` extension clicking on the `Switch to Pre-Release Version` in the mean time



> Sorry for the trouble, this should be fixed in 0.106.2

yes it  works, thanks!

**Comment by Akanixon** [comment]
I can confirm having the same problem across 20 clients now.
Switching to `Pre-Release` worked until today but now even that ain't working no more
